ui/app/mainui/SendModalHandler.qml

```diff
@@ -287,16 +300,22 @@ QtObject {
                 }
             }
 
-            // TODO: this should be called from the Reiew and Sign Modal instead
             onReviewSendClicked: {
-                root.transactionStoreNew.authenticateAndTransfer(backendHandler.uuid, selectedAccountAddress)
+                if (handler.selectedCollectibleEntry.available &&
+                        !!handler.selectedCollectibleEntry.item) {
+                    root.fnGetDetailedCollectible(simpleSendModal.selectedChainId ,
+                                                  handler.selectedCollectibleEntry.item.contractAddress,
+                                                  handler.selectedCollectibleEntry.item.tokenId)
+                }
+                Global.openPopup(sendSignModalCmp)
```

Comment:
@Khushboo-dev-cpp I don't see the relation between `fetchedPathModel` from `id: handler` and `sendSignModalCmp`? How do we know which tx (path) is a subject for reviewing (observed in the send sign modal)?

Reply:
What I expected here is to have a mechanism that iterates over the path modal (cautiously) and sing a single path (or approval) in each iteration. 

That way we will be covered for the following:
- bridge,
- multitx later,
- we follow the `fetchedPathModel`, cause it definitely may have more than a single path, otherwise it would not be a model (for example in `totalFeesAggregator` you've aggregated `txTotalFee` of each path, which is the correct approach)